exercises/nodejs/src/game.js

```diff
@@ -69,74 +68,74 @@ var Game = function () {
     if (currentCategory() == 'Rock') { console.log(rockQuestions.shift()) }
   }
 
+  this.movePlayer = function(roll) {
+    places[currentPlayer] = places[currentPlayer] + roll
+
+    if (places[currentPlayer] > 11) {
+      places[currentPlayer] = places[currentPlayer] - 12
+    }
+
+    console.log(players[currentPlayer] + "'s new location is " + places[currentPlayer])
+    console.log('The category is ' + currentCategory())
+  }
+
+  this.isInPenaltyBox = function() {
+    return inPenaltyBox[currentPlayer]
+  }
+
   this.roll = function (roll) {
     console.log(players[currentPlayer] + ' is the current player')
     console.log('They have rolled a ' + roll)
 
-    if (inPenaltyBox[currentPlayer]) {
+    if (this.isInPenaltyBox()) {
       if (roll % 2 != 0) {
         isGettingOutOfPenaltyBox = true
 
         console.log(players[currentPlayer] + ' is getting out of the penalty box')
-        places[currentPlayer] = places[currentPlayer] + roll
-        if (places[currentPlayer] > 11) {
-          places[currentPlayer] = places[currentPlayer] - 12
-        }
-
-        console.log(players[currentPlayer] + "'s new location is " + places[currentPlayer])
-        console.log('The category is ' + currentCategory())
+        
+        this.movePlayer(roll)
         askQuestion()
       } else {
         console.log(players[currentPlayer] + ' is not getting out of the penalty box')
         isGettingOutOfPenaltyBox = false
       }
     } else {
-      places[currentPlayer] = places[currentPlayer] + roll
-      if (places[currentPlayer] > 11) {
-        places[currentPlayer] = places[currentPlayer] - 12
-      }
-
-      console.log(players[currentPlayer] + "'s new location is " + places[currentPlayer])
-      console.log('The category is ' + currentCategory())
+      this.movePlayer(roll)
       askQuestion()
     }
   }
 
-  this.wasCorrectlyAnswered = function () {
-    if (inPenaltyBox[currentPlayer]) {
-      if (isGettingOutOfPenaltyBox) {
-        console.log('Answer was correct!!!!')
-        purses[currentPlayer] += 1
-        console.log(players[currentPlayer] + ' now has ' +
-            purses[currentPlayer] + ' Gold Coins.')
-
-        var winner = didPlayerWin()
-        currentPlayer += 1
-        if (currentPlayer == players.length) { currentPlayer = 0 }
-
-        return winner
-      } else {
-        currentPlayer += 1
-        if (currentPlayer == players.length) { currentPlayer = 0 }
-        return true
-      }
-    } else {
-      console.log('Answer was correct!!!!')
-
-      purses[currentPlayer] += 1
-      console.log(players[currentPlayer] + ' now has ' +
-          purses[currentPlayer] + ' Gold Coins.')
+  this.addGoldCoin = function() {
```

Comment:
Much clearer name (although I think the game would be clearer if the gold coins were just called score 😆 ).